portfolio/src/main/webapp/index.html

```diff
@@ -140,9 +140,9 @@ <h2 id="comments">Comments</h2>
         <div id="button-container">
           <button id="delete" onclick="deleteAllComments()">Delete All Comments</button>
           <div id="page-turn">
-			<button id="next" class="page-button" onclick="nextPage()">Next &raquo;</button>
-			<button id="previous" class="page-button" onclick="previousPage()">&laquo; Previous</button>
-		</div>
+			      <button id="next" class="page-button" onclick="nextPage()">Next &raquo;</button>
+			      <button id="previous" class="page-button" onclick="previousPage()">&laquo; Previous</button>
+		      </div>
```

Comment:
This may just be github rendering it weird, but check the alignment of the divs in this block

Reply:
Yeah that's weird, I think that it might be a github issue. It looks fine on my end: https://screenshot.googleplex.com/rgrDagwRG44